misc/org.projecthamster.hamster-gtk.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
```

Comment:
The application specific data are usually called resources. Though, I do not really care much, just pointing out the conventions I inferred.